src/utils/builder.ts

```diff
@@ -35,22 +36,42 @@ export function createNewPathway(name: string, description: string, pathwayId?:
 }
 
 export function downloadPathway(
-  pathway: Pathway,
+  pathway: Pathway | Pathway[],
   pathways: Pathway[],
   criteria: Criteria[],
   cpg = false
-): void {
-  const pathwayString = exportPathway(pathway, pathways, criteria, cpg);
-  // Create blob from pathwayString to save to file system
-  const pathwayBlob = new Blob([pathwayString], {
-    type: 'application/json'
-  });
+): Promise<void> {
+  // Determine if a single pathway or multiple pathways are being exported
+  const paths =
+    Array.isArray(pathway) && pathway.length > 1 ? pathway : ([] as Pathway[]).concat(pathway)[0];
```

Comment:
Is there a reason you use `([] as Pathway[]).concat(pathway)[0]` instead of just `pathway`. Seems like a very long way to get back what you started with....

It might also just be easier to make `pathway` always be a list and then just check the length in the if statement below.

Reply:
`([] as Pathway[]).concat(pathway)[0]` is used in the case where only a single pathway and wraps that pathway in an array.  I'd be fine with making pathway always be a list, but other existing code already expects to be able to pass in a single pathway and might need similar checks anyways.